scripts/generate_export.py

```diff
@@ -1,15 +1,16 @@
 import os
 import sys
+from indico.client.client import IndicoClient
 from tqdm import tqdm
 
 import pandas as pd
 from collections import defaultdict
-from solutions_toolkit.auto_review import Reviewer, FieldConfiguration
+# from solutions_toolkit.auto_review import Reviewer, FieldConfiguration
 from solutions_toolkit.uipath_block_scripts.config import ExportConfiguration
 from solutions_toolkit.indico_wrapper import IndicoWrapper
 
 
-USAGE_STRING = "USAGE: python3 generate_export path/to/configuration_file"
+USAGE_STRING = "USAGE: C:\\Users\\Gnana Peddi\\Documents\\Python Scripts\\configuration files\\Controllership_config_dev.yaml"
```

Comment:
Any particular reason for changing the USAGE string? This variable is only used when the program isn't called correctly.

Reply:
I was trying something out and forgot to change it back. Please ignore this as well.